operator/internal/manifests/var.go

```diff
@@ -78,8 +78,8 @@ const (
 	AnnotationLokiConfigHash string = "loki.grafana.com/config-hash"
 	// AnnotationLokiObjectStoreHash stores the last SHA1 hash of the loki object storage credetials.
 	AnnotationLokiObjectStoreHash string = "loki.grafana.com/object-store-hash"
-	// AnnotationLokiManagedAuthHash stores the last SHA1 hash of the loki managed auth credentials.
-	AnnotationLokiManagedAuthHash string = "loki.grafana.com/managed-auth-hash"
+	// AnnotationLokiTokenCCOAuthHash stores the last SHA1 hash of the loki token cco auth credentials.
```

Comment:
Does this improve the message?

```suggestion
// AnnotationLokiTokenCCOAuthHash stores the SHA1 hash of the secret generated by the Cloud Credential Operator.
```